user_guide_src/source/helpers/url_helper.rst

```diff
@@ -360,3 +360,25 @@ The following functions are available:
     Pass the URL string to the function like this::
 
         $url = prep_url('example.com');
+
+.. php:function:: url_to([$controller[, ...$args]])
```

Comment:
Based on the function signature, `$controller` is not an optional argument, so probably you can remove the square brackets around it.

Reply:
Oh yea, that was a mistake, fixing that!